mlflow/genai/judges/prompts/user_frustration.py

```diff
@@ -0,0 +1,66 @@
+# NB: User-facing name for the user frustration assessment.
+USER_FRUSTRATION_ASSESSMENT_NAME = "user_frustration"
+
+USER_FRUSTRATION_PROMPT = """\
```

Comment:
On a general note, we've found it's typically an anti-pattern to have long/complex prompts through rigorous evaluations on our existing judges. The more examples, cases, and generally text that is added, the more likely the judge will overfit, break in nuanced ways, or just not work in generic scenarios. We rely on the ability to tune judges to change these prompts for more specific use cases.

I would highly recommend simplifying as much as possible to a few sentences (IMO I think 2-3 sentences is the right amount for this metric), and generally following the patterns of the other judges - you can see other examples here: https://github.com/mlflow/mlflow/tree/cc26d9c2a5a05e2109fb5817f8b68ae1c74f79bb/mlflow/genai/judges/prompts